### ISSUE TITLE
Docs: add version badge for newest features

### ISSUE THREAD

**Issue by DirtyF** [jekyll/jekyll#6568]
```gherkin
Given a new Jekyll version is released
When jekyll users are browsing the documentation website
Then they can see what has been added in the latest version
```

Fix #6549 

Original props: https://github.com/jekyll/jekyll/issues/6549#issuecomment-344191986

Example:
![image](https://user-images.githubusercontent.com/103008/32993239-6e8f4506-cd55-11e7-8d13-601c60cc90b4.png)

**Comment by ashmaroli** [comment]
#### Some notes

- let's add this to sections / list-items / table-cells that document latest features.
- IMO, these tags should be removed when a new minor major version is released lest that docs get *cluttered* with numerous copies of these yellow labels.
- its best written as an include to allow entering versions via `include-params`

**Comment by pup** [comment]
i'd imagine this is going to require a bit of maintenance, as with every release, somebody is going to have to go through the docs and manually delete all old version badges, right?

**Comment by ashmaroli** [comment]
@pup, as I see it, in the *include* there would be an if block that ensures an output only if the *include-parameter-value* exceeds value from `{{ jekyll.version }}`

**Comment by DirtyF** [comment]
It's quiet normal to state since when a feature is available, those badges are a poor patch to hide the fact that we don't version our documentation.

**Comment by ashmaroli** [comment]
Resolved via #6574
